pages/common/funzip.md

```diff
@@ -0,0 +1,16 @@
+# funzip
+
+> Print the content of the first (non-directory) member in an archive without extraction.
+> More information: <https://manned.org/funzip>.
+
+- Print the content of the first member in a `.zip` archive:
+
+`funzip {{path/to/archive.zip}}`
+
+- Print the content in a `.gz` archive:
```

Comment:
```suggestion
- Print the content of the first member in a `.gz` archive:
```
Shouldn't this be the same as the first command description?

Reply:
Gzip can compress only a single file. Specifying 'first member' in this case is not ethical.